src/chttpd_handlers.erl

```diff
@@ -51,6 +54,20 @@ design_handler(HandlerKey, DefaultFun) ->
 %% Internal Function Definitions
 %% ------------------------------------------------------------------
 
+is_disabled(KeyBin) ->
+    BlacklistConfig = config:get("chttpd", "url_handler_blacklist", []),
```

Comment:
@janl If end user decided to black list something, they may (or need in this proposal) define a reason why, right? 

We can do this smarter, but smarter solutions takes more time/resources while that's stupid one just works and quite trivial in implementation. We can always improve it later. Especially, after collecting user experience for this blacklist management - may be they wouldn't touch it a lot (actually, they will) and that decision is good enough.